src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -36,6 +36,14 @@ class Install extends AbstractDatabase
      */
     protected $skip_check = false;
 
+
+    /**
+     * Flag to use an already existent database
+     *
+     * @var boolean
+     */
+    protected $existent_db = false;
```

Comment:
@levnikmyskin Instead of `existent_db`, could we call it `skip_create_statement` please? That makes a bit more sense.